morphocluster/frontend/src/views/Files.vue

```diff
@@ -113,6 +108,12 @@ export default {
         async initialize() {
             try {
                 this.entry = await api.getFileInfo(this.file_path);
+                this.breadcrumb = this.entry.path.split('/');
+                if (this.breadcrumb.includes(".")) {
+                    this.breadcrumb = []; // Oder eine andere Aktion, um das unerwünschte Element zu entfernen
+                }
+                this.breadcrumb = ["files"].concat(this.breadcrumb);
+                state.setBreadcrumbs(this.breadcrumb, "files");
             } catch (error) {
```

Comment:
This needs to be reworked.

`setBreadcrumbs()` takes a list of [router targets](https://router.vuejs.org/guide/essentials/navigation.html#Navigate-to-a-different-location) (see `Project.vue`).

To create the list of breadcrumbs (the list of all ancestor paths), you need to successively remove the last segment of the path and create a link.

The result needs to be something like:
```javascript
breadcrumbs = [
    {name: "files", params: {file_path: "/"}},
    {name: "files", params: {file_path: "/foo"}},
    {name: "files", params: {file_path: "/foo/bar"}},
    {name: "files", params: {file_path: "/foo/bar/baz"}},
    {name: "files", params: {file_path: "/foo/bar/baz/spam.txt"}},
]
this.setBreadcrumbs(breadcrumbs);
```